### PR TITLE
File Added : Changes Done

### PR DESCRIPTION
**Pull request by kalyani1942**

### CODE REVIEW

File: Kalyani/HTML/Assginment1/Assin.html [hunk 1]
```diff
@@ -1,18 +1,15 @@
...
+	    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+	    <title>AssignmentA1_1</title>
+	</head>
+	<body>
+	    <h1>HTML</h1>
+	    <h3>Introduction</h3>
+	    <p>HTML is a standard markup language for Web Pages.
+	    </p>
+	</body>
 </html>
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assginment1/Assin.html]: add new end line in all files.

File: Kalyani/HTML/Assginment1/Assin.html [hunk 2]
```diff
@@ -1,18 +1,15 @@
...
+	    <meta charset="UTF-8">
+	    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+	    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+	    <title>AssignmentA1_1</title>
+	</head>
+	<body>
+	    <h1>HTML</h1>
+	    <h3>Introduction</h3>
+	    <p>HTML is a standard markup language for Web Pages.
+	    </p>
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assginment1/Assin.html]: close the <p> tag on the same line

File: Kalyani/HTML/Assginment1/Assin2.html [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
+        <h3>Introduction</h3>
+        <p>HTML is a standard markup language for Web Pages.</p>
+        <h2>HTML editor</h2>
+        <h3>Notepad or Textedit</h3>
+        <p>
+            Web pages can be created and modified by using professional HTML editors. <br> However, 
+            for learning HTML we recommend a simple text editor like Notepad (PC) or TextEdit (Mac).
+        </p>
+    </body>
 </html>
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assginment1/Assin2.html]: add extra new line always in each file after </html>

File: Kalyani/HTML/Assginment1/Assin4.html [hunk 4]
```diff
@@ -1,7 +1,14 @@
...
-I am Kalyani Deshpande<br>
-I am a Software Enginner
+	<head>
+	    <title>
+	        <h2>Introduction</h2>
+	    </title>
+	</head>
+	<body style="background-color: lightblue">
+	    <p>
+	        <i><h2 style="color: red">I am Kalyani Deshpande</h2></i><br />I am a Software Enginner.
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assginment1/Assin4.html]: <i> is inline element and <h2> is block level element u can not include block level elements inside inline element.so do the opposite of what u have done here

File: Kalyani/HTML/Assginment1/Assin4.html [hunk 5]
```diff
@@ -1,7 +1,14 @@
...
+	        <b>I am from Pune</b>
+	    </p>
+	</body>
+</html>
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assginment1/Assin4.html]: add a end line in each file

File: Kalyani/HTML/Assginment3/KalyaniA3_1.html [hunk 6]
```diff
@@ -1,57 +1,53 @@
...
+                    <li>Dipti</li>
+                </ul>
+                <li>SYSTEM-2</li>
+                <ul type="disc">
+                    <li>Triveni</li>
+                    <li>Kalyani</li>
+                </ul>
+            </ol>
+    </body>
 </html>
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assginment3/KalyaniA3_1.html]: add new end line

File: Kalyani/HTML/Assginment3/KalyaniA3_2contact.html [hunk 7]
```diff
@@ -1,48 +1,43 @@
...
+                us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us. Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us. Contact us.Contact
+                us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us. Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us. Contact us.Contact
+                us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us. Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us. Contact us.Contact
+                us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us. Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us .Contact us.Contact
+            </p>
+        </div>
+    </body>
 </html>
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assginment3/KalyaniA3_2contact.html]: add new end line

File: Kalyani/HTML/Assignment2/A2.html [hunk 8]
```diff
@@ -1,36 +1,30 @@
...
-    </p>
-<center style="color: slategrey; font-size: 25px;">Thankyou!</center>
-</body>
-
-</html> 
+    <head>
+        <title>My first style page</title>
+    </head>
+    <body style="background-color: lightgreen">
+        <center>
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assignment2/A2.html]: dont use center tag instead use style text-align: center property on <h1> tag.

File: Kalyani/HTML/Assignment2/A2.html [hunk 9]
```diff
@@ -1,36 +1,30 @@
...
+            <h1>Google</h1>
+        </center>
+        <small><i>This article is about the company.</i></small>
+        <h2 style="font-size: 40" ;>his article is about the company.</h2>
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assignment2/A2.html]: remove the extra space before semicolon

File: Kalyani/HTML/Assignment2/A2.html [hunk 10]
```diff
@@ -1,36 +1,30 @@
...
+        <p>
+            Working from their dorm rooms, they built a search engine that used links to determine the importance of <br />individual pages on the <b style="color: darkcyan">World Wide Web</b>
+        </p>
+        <p>They called this search engine <i style="background-color: lightblue">Backrub</i>.</p>
+            <h1>Initial public offering</h1>
+        <p>
+            Google's initial public offering (IPO) took place five years later, on 19th August 2004. <br /> At that time Larry Page, Sergey Brin, and Eric Schmidt agreed to work together at Google for <del>30</del> <ins>20</ins> <br /> years, until the year
+            2024.
+        </p>
+        <center style="color: slategrey; font-size: 25px">Thankyou!</center>
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assignment2/A2.html]: dont use center tag instead use text-align: center property

File: Kalyani/HTML/Assignment4/KalyaniA4_1.html [hunk 11]
```diff
@@ -1,54 +1,50 @@
...
-        <tr>
-            <th>Female</th>
-            <td>1.7</th>
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Assignment4_1</title>
+        <style>
+            table,
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assignment4/KalyaniA4_1.html]: dont split table th td in different lines write it as like table, th, td {style u want to apply}.

File: Kalyani/HTML/Assignment4/KalyaniA4_2.html [hunk 12]
```diff
@@ -1,73 +1,69 @@
...
-        </tr>
-</body>
-
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Assignment4_2</title>
+        <style>
+            table,
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assignment4/KalyaniA4_2.html]: include tableth and td in a single line.

File: Kalyani/HTML/Assignment4/KalyaniA4_3.html [hunk 13]
```diff
@@ -1,72 +1,67 @@
...
-    </table>
-</body>
-
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Assignment4_3</title>
+        <style>
+            table,
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assignment4/KalyaniA4_3.html]: include all tags in a single line

File: Kalyani/HTML/Assignment4/KalyaniA4_3.html [hunk 14]
```diff
@@ -1,72 +1,67 @@
...
+            </tr>
+            <tr>
+                <th colspan="2">Schedule</th>
+                <th rowspan="2">Topic</th>
+            </tr>
+            <tr>
+                <th>Begin</th>
+                <th>End</th>
+            </tr>
+            </tr>
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assignment4/KalyaniA4_3.html]: remove this extra tr tag

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
do the changes as per comments given and extra new line at end in each file

File: Kalyani/HTML/Assignment4/KalyaniA4_1.html [hunk 15]
```diff
@@ -1,54 +1,48 @@
...
-        <tr>
-            <th>Female</th>
-            <td>1.7</th>
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Assignment4_1</title>
+        <style>
+            table,th,td {
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assignment4/KalyaniA4_1.html]: add space after comma

File: Kalyani/HTML/Assignment4/KalyaniA4_2.html [hunk 16]
```diff
@@ -1,73 +1,67 @@
...
-
-</html>
\ No newline at end of file
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Assignment4_2</title>
+        <style>
+            table,th,td {
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assignment4/KalyaniA4_2.html]: add a space after comma

File: Kalyani/HTML/Assignment4/KalyaniA4_3.html [hunk 17]
```diff
@@ -1,72 +1,64 @@
...
-
-</html>
\ No newline at end of file
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Assignment4_3</title>
+        <style>
+            table,th,td {
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assignment4/KalyaniA4_3.html]: add a space after comma always in all files

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
do changes as per comments given

**Review by SnehaMakade — APPROVED** [review]

File: Kalyani/HTML/Assginment3/KalyaniA3_2main.html [hunk 18]
```diff
@@ -1,55 +1,50 @@
...
-        a:visited {
-            color: blue;
-        }
-    </style>
-</head>
-
-<body>
-    <div style="text-align: center">
-        <h1 style="color: black;">Ambersoft</h1>
+    <head>
```
Comment (Pawan1005) [inline Kalyani/HTML/Assginment3/KalyaniA3_2main.html:3]: html tag is missing here

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**